src/blazar_tempest_plugin/tests/api/base.py

```diff
@@ -136,6 +136,6 @@ def _create_reserved_container(self, name, hints, desired_state="Running"):
             name=data_utils.rand_name(name),
             hints=hints,
             image="busybox",
-            command="/bin/sh -c 'echo hello-from-container && sleep 60'",
+            command="/bin/sh -c 'sleep infinity'",
```

Comment:
We do check that the containers "said something" when gathering the log output here:
https://github.com/ChameleonCloud/chi-blazar-tempest-plugin/blob/main/src/blazar_tempest_plugin/tests/api/test_device_containers.py#L66